README.md

```diff
@@ -28,6 +28,17 @@ Usage
     * <kbd>CTRL-G</kbd><kbd>CTRL-R</kbd> for **R**emotes
     * <kbd>CTRL-G</kbd><kbd>CTRL-H</kbd> for commit **H**ashes
     * <kbd>CTRL-G</kbd><kbd>CTRL-S</kbd> for **S**tashes
+  > :warning: You may have issues with these bindings in the following cases:
+  >
+  > * <kbd>CTRL-G</kbd><kbd>CTRL-B</kbd> will not work if
+  >   <kbd>CTRL-B</kbd> is used as the tmux prefix (bindkey)
```

Comment:
I didn't understand the "(bindkey)" at the end, does that refer to tmux or zsh?

Reply:
Oh, I'm gonna remove it.

Reply:
```suggestion
  >   <kbd>CTRL-B</kbd> is used as the tmux prefix
```